app/views/reporting/user_reports/index.html.erb

```diff
@@ -22,6 +22,17 @@
       </h3>
     <% end %>
 
+    <h2 class='govuk-heading-m'>Latest complete weekly reports<h2>
```

Comment:
Do you want to use an `i18n` here?

Reply:
This page is very much a test landing page so that Analysts / testers can get to the report under development. I imagine it will be very different once interaction designers have looked at it.